web/geolocation/geolocation_contants.js

```diff
@@ -0,0 +1,2 @@
+const geoChannelName = "Geolocation Channel";
+const devAPIKeyPrivate = "AIzaSyDyl7ItKt5viBBju5Rwsqwrii5soyWUzp0";
```

Comment:
Are you sure you want to check this in?? 
Also, please pull this value from /shared/keys.js (I imagine this working similar to the delivery application eventually, https://github.com/PubNubDevelopers/delivery-demo/blob/main/js/keys.js)